components/ArticleCard.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function ArticleCard({ imageUrl }) {
+  return (
+    <StyledListItem>
+      <Image src={imageUrl} width={600} height={400}></Image>
+    </StyledListItem>
+  );
+}
+
+const StyledListItem = styled.li`
+  height: 400px;
```

Comment:
you don't need this styling when the height of the image is already 400px

Reply:
ah thank you. I set the height for the text later. Those images will be removed. I could not resist to see them on the front page though :D:D